java/src/main/java/com/powsybl/dataframe/network/NetworkDataframes.java

```diff
@@ -126,6 +127,56 @@ private static MinMaxReactiveLimits getMinMaxReactiveLimits(ReactiveLimitsHolder
         return reactiveLimits instanceof MinMaxReactiveLimits ? (MinMaxReactiveLimits) reactiveLimits : null;
     }
 
+    private static <U extends Injection<U>> BooleanSeriesMapper.BooleanUpdater<U> connectInjection() {
+        return (g, b) -> {
+            if (b) {
+                g.getTerminal().connect();
```

Comment:
could simplifier

Reply:
how would you do that ? 
this does not work  :
return (g, b) -> { b ? g.getTerminal().connect() : g.getTerminal().disconnect();};

Reply:
it works with 
return (g, b) -> { Boolean res = b ? g.getTerminal().connect() : g.getTerminal().disconnect();};